assets/components/checkout/checkout.jsx

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import React, { type Node } from 'react';
+
+import './checkout.scss';
+
+
+type PropTypes = {
+  children: Node,
+};
+const Checkout = ({ children }: PropTypes) => (
+  <div className="component-checkout">
+    <div className="component-checkout__form">
+      {children}
+    </div>
+  </div>
+);
+
+export default Checkout;
```

Comment:
this component makes little sense now but it'll contain way more stuff (it's the checkout layout) down the line so it's got room to grow